### PR TITLE
Locality attribute

### PR DESCRIPTION
**Pull request by gares**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -15,6 +15,31 @@
 ### API
 - New `coq.arguments.set-default-implicit` that behaves like
   `Arguments foo : default implicits`
+- Change of arguments of type `@global?` attributes `@local!` or `@global!`.
+  In order to pass a locality directive one has to do something like
+    `@global! => coq.add-something`
```
Comment (CohenCyril) [inline Changelog.md:20]: Is the semantics of `@global! => (@local! => stuff)` well defined?

Comment (CohenCyril) [reply Changelog.md:20]: (I think I remember you telling me that `@global!` would win, but I find it counterintuitive)

Comment (gares) [reply Changelog.md:20]: Hum, local should win.

Comment (gares) [reply Changelog.md:20]: I'll add a test

Comment (gares) [reply Changelog.md:20]: What may be unclear to a user is `[ @local! , @global! ] =>foo1` (local wins), but it looks artificial

**Pull request merged**